packages/react-components/react-nav-preview/library/src/components/AppItem/useAppItemStyles.styles.ts

```diff
@@ -25,11 +38,24 @@ const useStyles = makeStyles({
 export const useAppItemStyles_unstable = (state: AppItemState): AppItemState => {
   'use no memo';
 
-  const styles = useStyles();
-  state.root.className = mergeClasses(appItemClassNames.root, styles.root, state.root.className);
+  const rootDefaultClassName = useRootDefaultClassName();
+  const iconStyles = useIconStyles();
+  const appItemSpecificStyles = useStyles();
+
+  const { size, isIconPresent } = state;
+
+  state.root.className = mergeClasses(
+    appItemClassNames.root,
+    rootDefaultClassName,
```

Comment:
I think rootDefaultClassName should go before appItemClassNames.root so that users can modify the underlying base styles via the more specific root className

Reply:
got it - changed!